### ISSUE TITLE
"You seem to not be depending on "@angular/core". This is an error." after npm start

### ISSUE THREAD

**Issue by Muscan** [bondar-artem/ngx-cypress-test#43]
After the npm start, this error is displayed:
"PS D:\GitRepo\ngx-cypress-test> npm start

> ngx-admin@4.0.1 start
> ng serve

? Would you like to share pseudonymous usage data about this project with the Angular Team
at Google under Google's Privacy Policy at https://policies.google.com/privacy. For more  
Thank you for sharing pseudonymous usage data. Should you change your mind, the following
command will disable this feature entirely:

    ng analytics disable

Global setting: not set
Local setting: enabled
Effective status: enabled
DEPRECATED: The 'defaultProject' workspace option has been deprecated. The project to use will be determined from the current working directory.   
You seem to not be depending on "@angular/core". This is an error.
PS D:\GitRepo\ngx-cypress-test>
PS D:\GitRepo\ngx-cypress-test> npm start

> ngx-admin@4.0.1 start
> ng serve

DEPRECATED: The 'defaultProject' workspace option has been deprecated. The project to use will be determined from the current working directory.
You seem to not be depending on "@angular/core". This is an error."

![image](https://user-images.githubusercontent.com/48117818/226735756-f516bd54-b73f-4d52-9bad-3c87e3de2c9d.png)

Basically I cannot use the application further and go further with the course from Udemy.

**Comment by bondar-artem** [comment]
Follow these steps:
1. Verify that you installed Node 16 and npm is version 8
2. Run `npm cache clean --force`
3. Delete global node_modules on Windows. It should be located on this path
C:\Users\<YourUserName>\AppData\Roaming\npm\node_modules
4. Restart the computer
5. Delete the project folder and clone it again
6. Run `npm install` if it will not work try` npm install --force` or `npm install --legacy-peer-deps`
7. Run `npm start`

**Comment by Muscan** [comment]
Now on another machine I`m getting this error at the same point, npm start. 
Versions: node v18.12.1,  npm 9.1.2

I mention that I also have other projects on the machine, which are running against IIS and I cannot delete the node _modules since I`m risking my other personal projects. 

![image](https://user-images.githubusercontent.com/48117818/226834516-61c59568-e6be-4cf4-b123-e6548d405fa0.png)

**Comment by Muscan** [comment]
Remembered you mentioned installing an older version of nodeJs
https://nodejs.org/download/release/v16.19.1/
This fixed the issue. 

My question is, if I have two applications, and one works with an older and the other with a newer version, how can I do that?

I will close this issue. 
My suggestion is that the dependencies  should be updated to the latest version.

**Comment by bondar-artem** [comment]
You can use nvm to switch between Node versions
